packages/cli/src/lib/colors.js

```diff
@@ -14,8 +14,16 @@ import chalk from 'chalk'
 export default {
   error: chalk.bold.red,
   warning: chalk.keyword('orange'),
-  green: chalk.green,
+  success: chalk.greenBright,
```

Comment:
I was having issues with this, but maybe related to rwfw. Just got `colours.success` is not a function in the final step of the setup command

Unsure why!